hack/move-upstream.sh

```diff
@@ -35,7 +37,14 @@ if [[ -n "${1-}" ]]; then
 fi
 
 echo "++ Generating patch for ${selector} onto ${lastrev} ..." 2>&1
-git diff -p --raw --relative=Godeps/_workspace/src/${repo}/ "${selector}" -- Godeps/_workspace/src/${repo}/ > "${patch}"
+index=0
+for commit in $(git log --no-merges --format="%H" --reverse "${selector}" -- "Godeps/_workspace/src/${repo}/"); do
+  git format-patch --raw --start-number=${index} --relative="Godeps/_workspace/src/${repo}/" "${commit}^..${commit}" -o "${patch}"
+  let index+=10
+done
+
+# remove all commits that had no entries
+find "${patch}" -type f -size 0 -exec rm {} \;
```

Comment:
`xargs` is a hammer most people don't really think belongs
One caveat to using `find` with `-exec` is that it will spawn a subshell to do  every command, so if this matches ten thousand things expect ten thousand subshells.